components/test_storage/src/assert_storage.rs

```diff
@@ -231,11 +234,15 @@ impl<E: Engine> AssertionStorage<E> {
 
     fn expect_not_leader_or_stale_command(&self, err: storage::Error) {
         match err {
-            storage::Error::Txn(txn::Error::Mvcc(mvcc::Error::Engine(kv::Error::Request(
-                ref e,
+            StorageError(box StorageErrorInner::Txn(TxnError(box TxnErrorInner::Mvcc(
+                MvccError(box MvccErrorInner::Engine(KvError(box KvErrorInner::Request(ref e)))),
+            ))))
```

Comment:
It's too long. Is there any way to simplify the error code? Such as use `use Box<ErrorInner> as Error;` instead of  `pub struct Error(pub Box<ErrorInner>);`

Reply:
I hope we can have some macros to simply it in future.